src/bilder/images/edxapp/files/migrations.service

```diff
@@ -0,0 +1,17 @@
+[Unit]
+Description="Obtain an exclusive lock and run database migrations at startup."
+Requires=network-online.target
+After=consul-template.service
+
+[Service]
+Type=oneshot
+ExecStart=/usr/local/bin/consul lock -timeout=1h -verbose lsm_migrations /edx/bin/edxapp-migrate-lms
+ExecStart=/usr/local/bin/consul lock -timeout=1h -verbose cms_migrations /edx/bin/edxapp-migrate-cms
```

Comment:
It would be best to actually serialize the LMS and CMS migrations because they have a number of shared models, so if you run all of the LMS migrations first there may not actually be any CMS migrations that need to run.

Reply:
This SO page seems to suggest these will run serially: https://stackoverflow.com/questions/48195340/systemd-with-multiple-execstart as does this redhat blog post: https://www.redhat.com/sysadmin/systemd-oneshot-service under "Chaining commands in a oneshot service" but I can't actually find a statement in the systemd documentation specifying it.

Reply:
Another SO post supporting that statement: https://stackoverflow.com/questions/39032100/what-is-the-difference-between-systemds-oneshot-and-simple-service-types